src/experimental/DataBase.h

```diff
@@ -43,7 +38,13 @@ class DataBase {
     /*!
      * \brief closes the database connection.
      */
-    bool disconnect();
+    void disconnect();
+
+    /*!
+     * \brief Can be used to access the database connection.
+     * \return The QSqlDatabase object pertaining to the connection.
+     */
+    QSqlDatabase database();
```

Comment:
you call this fucntion statically at Database::disconnect. since it returns something handled by Qt shouldnt it be static?